omero/autogen_docs

```diff
@@ -33,10 +33,14 @@ echo "Generating DB script example"
 (cd $WORKSPACE && OMERO.server/bin/omero db script --password secretpassword 2>&1) | sed "s|$WORKSPACE|/home/omero|" > omero/downloads/cli/db-script-example.txt
 
 echo "Generating Web configuration templates"
-$WORKSPACE/OMERO.server/bin/omero web config nginx | sed "s|$WORKSPACE|/home/omero|" > omero/sysadmins/unix/nginx-omero.conf
-$WORKSPACE/OMERO.server/bin/omero web config apache | sed "s|$WORKSPACE|/home/omero|" > omero/sysadmins/unix/apache-omero.conf
-$WORKSPACE/OMERO.server/bin/omero web config apache-fcgi | sed "s|$WORKSPACE|/home/omero|" > omero/sysadmins/unix/apache-fcgi-omero.conf
-
+$WORKSPACE/OMERO.server/bin/omero web config nginx | sed "s|$WORKSPACE|/home/omero|" > omero/sysadmins/unix/install-web/nginx-omero.conf
+$WORKSPACE/OMERO.server/bin/omero web config apache | sed "s|$WORKSPACE|/home/omero|" > omero/sysadmins/unix/install-web/apache-omero.conf
+$WORKSPACE/OMERO.server/bin/omero web config apache-fcgi | sed "s|$WORKSPACE|/home/omero|" > omero/sysadmins/unix/install-web/apache-fcgi-omero.conf
+$WORKSPACE/OMERO.server/bin/omero config set omero.web.application_server "wsgi"
+$WORKSPACE/OMERO.server/bin/omero web config apache-wsgi | sed "s|$WORKSPACE|/home/omero|" > omero/sysadmins/unix/install-web/apache-wsgi-omero.conf
```

Comment:
This sed transformation will need more work. Running locally:
```diff
diff --git a/omero/sysadmins/unix/install-web/apache-wsgi-omero.conf b/omero/sys
admins/unix/install-web/apache-wsgi-omero.conf
index 2f8c3f1..3e3629d 100644
--- a/omero/sysadmins/unix/install-web/apache-wsgi-omero.conf
+++ b/omero/sysadmins/unix/install-web/apache-wsgi-omero.conf
@@ -6,7 +6,7 @@
 #  OmeroForceSSL - redirect all http requests to https

 ###
-### Example SSL stanza for OMERO.web created 2015-07-23 12:16:08.997137
+### Example SSL stanza for OMERO.web created 2015-07-24 11:45:27.241834
 ###

 # Eliminate overlap warnings with the default ssl vhost
@@ -49,14 +49,14 @@
 #</VirtualHost>

 ###
-### Stanza for OMERO.web created 2015-07-23 12:16:08.997137
+### Stanza for OMERO.web created 2015-07-24 11:45:27.241834
 ###

 <VirtualHost _default_:80>

   DocumentRoot /home/omero/OMERO.server/lib/python/omeroweb

-  WSGIDaemonProcess omeroweb processes=5 threads=1 display-name=%{GROUP} user=omero python-path=/usr/lib/python2.7/site-packages/Ice:/home/omero/OMERO.server/lib/python:/home/omero/OMERO.server/lib/fallback:/home/omero/OMERO.server/lib/python/omeroweb
+  WSGIDaemonProcess omeroweb processes=5 threads=1 display-name=%{GROUP} user=simon python-path=/usr/local/lib/python2.7/site-packages:/home/omero/OMERO.server/lib/python:/Users/simon/work/ome-documentation/OMERO.server/lib/fallback:/Users/simon/work/ome-documentation/OMERO.server/lib/python/omeroweb

   WSGIProcessGroup omeroweb

```
- Multiple `$WORKSPACE` substitutions are needed on the same line- pass the `g` flag to sed
- The Python paths needs transforming